doc/src/MainPage.dox

```diff
@@ -235,13 +235,12 @@
  * rules for the manager:
  *
  * - Store and recall a @ref primary_string
- * - Store and recall @ref attributes
  * - Allow filtering by @ref Specification
```

Comment:
- Just above it says "three simple rules", which is just two now?
- We mention primary strings here only, should this be something more generic? "Data"?

Reply:
Good spot, thanks, will fix the count. Primary string will be removed in #269, as it still exists as of this PR.

Reply:
Updated text [here](https://github.com/TheFoundryVisionmongers/OpenAssetIO/pull/308/commits/939765c1b398a9facad409592dddf906752bcec8).